### PR TITLE
Fix Kupmios provider errors on protocol params

### LINKED ISSUE #61: Errors on Kupmios provider getProtocolParameters
When using the Kupmios provider, calls to getProtocolParameters raise errors.

### PR DESCRIPTION
**Pull request by caike**
Fix Kupmios errors when getting protocol parameters by updating it to work with the current major version of Ogmios.

Tested against the following versions:

Cardano Node 8.9.0
Ogmios 6.1.0

Fixes #61

### CODE REVIEW

File: packages/provider/src/kupmios.ts [hunk 1]
```diff
@@ -31,41 +31,38 @@ export class Kupmios implements Provider {
...
+              poolDeposit: BigInt(result.stakePoolDeposit.ada.lovelace),
               priceMem: parseInt(memNum) / parseInt(memDenom),
               priceStep: parseInt(stepsNum) / parseInt(stepsDenom),
               maxTxExMem: BigInt(result.maxExecutionUnitsPerTransaction.memory),
-              maxTxExSteps: BigInt(
-                result.maxExecutionUnitsPerTransaction.steps,
-              ),
-              coinsPerUtxoByte: BigInt(result.coinsPerUtxoByte),
+              maxTxExSteps: BigInt(result.maxExecutionUnitsPerTransaction.cpu),
+              coinsPerUtxoByte: BigInt(0),
```
Comment (caike) [inline packages/provider/src/kupmios.ts:65]: Maybe remove `coinsPerUtxoByte` altogether since this is no longer used for fee calculation ?

Comment (solidsnakedev) [reply packages/provider/src/kupmios.ts:65]: I prefer to keep it as it is because, even though we don't currently use it, there may be future use cases.

**Comment by solidsnakedev** [comment]
if you run `pnpm changeset` , I can take this PR
make sure it is a patch and write a small comment of the change

the changeset file should look like this

```
---
"@lucid-evolution/provider": patch
---

Fix Kupmios provider errors on protocol params
```

more info
https://github.com/changesets/changesets/blob/main/docs/adding-a-changeset.md

File: packages/provider/src/kupmios.ts (hunk 1 above)
Comment (solidsnakedev) [reply packages/provider/src/kupmios.ts:65]: sorry I meant to bring back `coinsPerUtxoByte: BigInt(result.coinsPerUtxoByte),`

Comment (caike) [reply packages/provider/src/kupmios.ts:65]: Oh I see, but this property is no longer part of protocol params. Unless I'm missing something, min fee calculation now only relies on `result.minFeeCoefficient` and `result.minFeeConstant.ada.lovelace`

Comment (solidsnakedev) [reply packages/provider/src/kupmios.ts:65]: Weird Ogmios is not following the cddl specs, https://github.com/IntersectMBO/cardano-ledger/blob/be59c234150b6c5a24596ec6829810c2d2dfa732/eras/babbage/impl/cddl-files/babbage.cddl#L270-L292

we can set `coinsPerUtxoByte: BigInt(0),` to zero for now

**Review by solidsnakedev — APPROVED** [review]

File: packages/utils/CHANGELOG.md
Comment (solidsnakedev) [inline packages/utils/CHANGELOG.md:1]: we don't need this file

File: packages/wallet/package.json
Comment (solidsnakedev) [inline packages/wallet/package.json:1]: we don't need this file

File: packages/wallet/CHANGELOG.md
Comment (solidsnakedev) [inline packages/wallet/CHANGELOG.md:1]: we don't need this file

File: packages/utils/package.json
Comment (solidsnakedev) [inline packages/utils/package.json:1]: we don't need this file

File: packages/provider/package.json
Comment (solidsnakedev) [inline packages/provider/package.json:1]: we don't need this file

File: packages/provider/CHANGELOG.md
Comment (solidsnakedev) [inline packages/provider/CHANGELOG.md:1]: we don't need this file

File: packages/lucid/package.json
Comment (solidsnakedev) [inline packages/lucid/package.json:1]: we don't need this file

File: packages/lucid/CHANGELOG.md
Comment (solidsnakedev) [inline packages/lucid/CHANGELOG.md:1]: we don't need this file

**Comment by solidsnakedev** [comment]
basically we just need `packages/provider/src/kupmios.ts` and the changeset file, I think it is better to sync from main and start again

File: packages/provider/src/kupmios.ts (hunk 1 above)
Comment (caike) [reply packages/provider/src/kupmios.ts:65]: I think I found it. The underlying property name was changed to [utxoCostPerByte](https://github.com/IntersectMBO/cardano-node/pull/4141) which is exactly what's [returned by the ledger](https://github.com/IntersectMBO/cardano-cli/blob/4bdbc16a02e4afa5b5b48f032a9de3e2ff3def51/cardano-cli/test/cardano-cli-golden/files/input/babbage/transaction-calculate-min-fee/protocol-params.json#L384) (matches the result from `cardano-cli query protocol-parameters`). Ogmios decided to rename this to [minUtxoDepositCoefficient](https://github.com/CardanoSolutions/ogmios/blob/83065af5d370389c09e901c64628781ac0898a03/architectural-decisions/accepted/017-api-version-6-major-rewrite.md?plain=1#L342) which does have a value of `4310`. I'll close this PR due to to conflicts and make these changes on the new one.

**Pull request closed without merging**